Makefile

```diff
@@ -102,7 +102,7 @@ k8s-e2e: ## Run k8s specific e2e test
 		-ginkgo.v
 
 .PHONY: test-e2e
-validate-e2e: ## Run e2e validation/gating test
+test-e2e: ## Run e2e validation/gating test
```

Comment:
this does not belong to this commit. Is there a counterpart PR for prown?

Reply:
ok, I will seperate it.

Reply:
Done through https://github.com/openshift/cluster-api-provider-aws/pull/136